.trunk/configs/.remarkrc.yaml

```diff
@@ -0,0 +1,4 @@
+plugins:
```

Comment:
Do you intend for this to be an incomplete subset of the plugins you specified in extra_packages?

Reply:
Yes - the file-reporter is for internal output not plugin usage. I did remove the tabs one - its not needed